registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -624,6 +627,7 @@
     "sha256:929d39dcc639efcd63572acff37137e591451f57c0338cf05605b3983a4a672c": ["v1.10.3"]
     "sha256:713536111be9ebd21f1aef8bb561df611b1d649fb22bcadd5f9b0e97ca272d41": ["v1.11.0-alpha.0"]
     "sha256:638d61de0a08b6fc11fb5adea3c805e3e0621da0c666199308825ac9ff01d77e": ["v1.11.0-alpha.1"]
+    "sha256:78c7f9174906ae6329252c6008c746f01b277af7095ae9f7ccd19c7b8d4bb853": ["v1.11.0-alpha.2"]
```

Comment:
SHA `78c7f9174906` matches kubadm-control-plane-controller v1.11.0-alpha.2 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-control-plane-controller@sha256:78c7f9174906ae6329252c6008c746f01b277af7095ae9f7ccd19c7b8d4bb853/details?inv=1&invt=Ab1mPQ). 👍🏻